internal/clients/telegram/message.go

```diff
@@ -36,12 +39,45 @@ type message struct {
 	Text           *string  `json:"text,omitempty"`
 }
 
+// Generated a sendMessage with ChatID == message.Chat.ID
+func (m *message) sameChatPlain(text string) sendMessage {
+	return sendMessage{
+		ChatID:    strconv.FormatInt(m.Chat.ID, 10),
+		Text:      text,
+		ParseMode: "",
+	}
+}
+
+func (m *message) sameChatMarkdownV2(text string) sendMessage {
+	return sendMessage{
+		ChatID:    strconv.FormatInt(m.Chat.ID, 10),
+		Text:      text,
+		ParseMode: "MarkdownV2",
+	}
+}
+func (m *message) sameChatHtml(text string) sendMessage {
+	return sendMessage{
+		ChatID:    strconv.FormatInt(m.Chat.ID, 10),
+		Text:      text,
+		ParseMode: "html",
+	}
+}
+
 type callbackQuery struct{}
-type user struct{}
+type user struct {
+	Id int64 `json:"id"`
```

Comment:
In uppercase please: ID

Reply:
https://github.com/m-kuzmin/daily-reporter/pull/4/commits/74ea3a2498083efa119bda05cfcf5e8db8e589c1